projects/fal/src/fal/cli/deploy.py

```diff
@@ -95,16 +98,11 @@ def _deploy(args):
     )
 
     if app_id:
-        gateway_host = _remove_http_and_port_from_url(host.url)
-        gateway_host = (
-            gateway_host.replace("api.", "").replace("alpha.", "").replace("ai", "run")
-        )
-
         args.console.print(
             "Registered a new revision for function "
             f"'{app_name}' (revision='{app_id}')."
         )
-        args.console.print(f"URL: https://{gateway_host}/{user_id}/{app_name}")
+        args.console.print(f"URL: https://fal.ai/models/{user.username}/{app_name}")
```

Comment:
Previewsly we took previews/shark into account, but now the URL is always a prod one. And if one need an app URL, it will have to be extracted from the playground page (in API tab), right?

Reply:
That is correct. I'm not sure if we have that information now, but I can quickly implement it. Another option was to print both the "run" url and the "playground" url, but I'm afraid that would be a bit confusing.

Reply:
For the record: discussed with @mederka that both Endpoint and Playground are useful to have around. For now the playground one doesn't reflect previews/shark, but that's fine for developers until we add some query param support in web-app in the mid-future.